acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/BesuNodeRunner.java

```diff
@@ -54,4 +54,19 @@ default void waitForPortsFile(final Path dataDir) {
               }
```

Comment:
Intellij is telling me the above method is unused. It looks like a more specialized version of the below. Can it be removed?

Reply:
yes you are right I just removed it